Modules/Poll/classes/class.ilObjPollGUI.php

```diff
@@ -182,6 +182,8 @@ protected function updateCustom(ilPropertyFormGUI $a_form)
 		}
 		else
 		{
+			$this->object->setVotingPeriodBegin(0);

```

Comment:
I think setVotingPeriodBegin/End(null) is more precise here. 
The values are store in table il_poll as unix timestamp. 
null means no date
0 means 1970-01-01 00:00:00